src/main/java/net/sf/mzmine/modules/masslistmethods/listexport/help/help.html

```diff
@@ -0,0 +1,98 @@
+<html>
```

Comment:
Was this added in the MZminePI code? It doesn't look like anything Dan would have written, but it seems to be help for the ```listexport``` module that he wrote.